inc/class-upgrade.php

```diff
@@ -725,11 +726,29 @@ private function upgrade_145() {
 	 * Performs the 14.9 upgrade.
 	 */
 	private function upgrade_149() {
-		$version = get_option( 'wpseo_license_server_version', WPSEO_License_Page_Manager::VERSION_BACKWARDS_COMPATIBILITY );
+		$version = get_option( 'wpseo_license_server_version', 2 );
 		WPSEO_Options::set( 'license_server_version', $version );
 		delete_option( 'wpseo_license_server_version' );
 	}
 
+	/**
+	 * Performs the 15.1 upgrade
+	 */
+	private function upgrade_151() {
```

Comment:
Not sure I understand what this upgrade method does. Could you please explain?

Reply:
I removes the notifications set when a user has an old version of the plugin. This is because the older versions calls the old edd endpoint. 

I think we should remove this logic. Do you agree?